com.unity.shadergraph/Editor/Data/Graphs/Texture2DShaderProperty.cs

```diff
@@ -27,7 +45,8 @@ internal Texture2DShaderProperty()
 
         internal override string GetPropertyBlockString()
         {
-            return $"{hideTagString}{modifiableTagString}[NoScaleOffset]{referenceName}(\"{displayName}\", 2D) = \"{defaultType.ToString().ToLower()}\" {{}}";
+            var normalTagString = (defaultType == DefaultType.NormalMap) ? "[Normal]" : "";
+            return $"{hideTagString}{modifiableTagString}{normalTagString}[NoScaleOffset]{referenceName}(\"{displayName}\", 2D) = \"{ToShaderLabString(defaultType)}\" {{}}";
```

Comment:
Any Texture property tagged as "Bump/NormalMap" now adds the [Normal] tag to help ensure users give it a normal map texture.